src/shared/ui/inputHeader/InputHeader.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { Meta, StoryFn } from '@storybook/react';
+import { useState } from 'react';
+import InputHeader, { InputHeaderProps } from '@/shared/ui/inputHeader/InputHeader';
+import { MultiSelect, SelectItem } from '@/shared/ui/select/MultiSelect';
+import { Select, SelectListItem } from '@/shared/ui/select/Select';
+import TextField from '@/shared/ui/textfield/TextField';
+
+const SELECT_ITEM = [
+  { id: 1, label: '1-2년차' },
+  { id: 2, label: '3-5년차' },
+  { id: 3, label: '5-7년차' },
+];
+const MULTI_SELECT = [
+  { id: 1, item: 'Svelte' },
+  { id: 2, item: 'Bootstrap' },
+  { id: 3, item: 'NestJs' },
+  { id: 4, item: 'Express' },
+  { id: 5, item: 'Django' },
+  { id: 6, item: 'Spring Boot' },
+  { id: 7, item: 'Laravel' },
+  { id: 8, item: 'Fast API' },
+  { id: 9, item: 'Flask' },
+  { id: 10, item: 'Node.js' },
+  { id: 11, item: 'MySQL' },
+  { id: 12, item: 'PostgreSQL' },
+  { id: 13, item: 'Docker' },
+  { id: 14, item: 'Ruby on Rails' },
+  { id: 15, item: 'ElasticSearch' },
+  { id: 16, item: 'Terraform' },
+  { id: 17, item: 'GraphQL' },
+];
+
+const meta: Meta<typeof InputHeader> = {
+  title: 'shared/InputHeader',
```

Comment:
```suggestion
  title: 'component/shared/InputHeader',
```
로 경로를 설정하는 편이 좋을 거 같습니다~

Reply:
수정했습니다! 다른 컴포넌트들도 한꺼번에 바꾸겠습니다!